src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryTimeTowerManager.kt

```diff
@@ -78,7 +78,8 @@ object ChocolateFactoryTimeTowerManager {
         if (!isTimeTowerActive && wasTimeTowerRecentlyActive && config.timeTowerReminder && currentCharges() > 0) {
             val charges = StringUtils.pluralize(currentCharges(), "charge", "charges", withNumber = true)
             ChatUtils.clickableChat(
-                "§cYour Time Tower just expired and has $charges remaining. Click here to use one.",
+                "§cYour Time Tower just expired and has $charges remaining." +
```

Comment:
```suggestion
                "§cYour Time Tower just expired and has $charges remaining. " +
```

Reply:
is this meant to be on a new line or are you just formatting these 2 lines for fun?